src/Main.js

```diff
@@ -53,22 +54,39 @@ class Main extends Component {
   // get list of playlists and playlist list length. also attach invert event
   componentWillMount() {
     window.addEventListener('keydown', (e) => this.handleInvert(e))
-
-    const { activePage, per } = this.state
-
     Promise.all([
       this.API.getBlockCount(),
-      this.API.getPaginatedChannelContents(activePage, per),
+      this.API.getChannelContents(),
     ])
-      .then(([length, playlists]) => {
+      .then(([length, playlistChannel]) => {
+        const resortedPlaylists = {
+          ...playlistChannel,
+          contents: playlistChannel.contents.reverse()
+        }
         this.setState({
           playlistListLength: length,
-          playlists,
-          maxItemsInCurrentPage: this.getMaxItemsInCurrentPage(length, this.state.per),
+          playlistChannel: resortedPlaylists,
+          searchList: playlistChannel,
```

Comment:
😎

Reply:
hmm i'm not sure searchlist is actually used now that i see this